src/main/resources/mcmod.info

```diff
@@ -3,7 +3,7 @@
     "modid": "landofsignals",
     "name": "LandOfSignals",
     "description": "Adds fancy railroad signals, signs, decor and other stuff to the game. Use best in combination with a train mod.",
-    "version": "1.0.2",
+    "version": "1.0.3",
```

Comment:
I'll add this to the release pipeline too. So we'll just need to change it in VERSION